### PR TITLE
Feature/chart/project

### PR DESCRIPTION
**Pull request by karolcienkosz**

### CODE REVIEW

File: charts/project-config-connector/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: project-config-connector
 description: GCP project
 type: application
-version: 0.0.1
+version: 0.1.1
```
Comment (michalbagrowski) [inline charts/project-config-connector/Chart.yaml]: 0.1.0?

File: charts/project-config-connector/templates/project.yaml [hunk 2]
```diff
@@ -42,4 +42,28 @@ spec:
...
+
+{{- if $.Values.services }}
+{{- range $service := .Values.services.list  }}
+---
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  annotations:
+    {{- if $.Values.services.protection }}
+    cnrm.cloud.google.com/deletion-policy: {{ $.Values.services.protection.deletionPolicy | default "abandon"  | quote }}
```
Comment (michalbagrowski) [inline charts/project-config-connector/templates/project.yaml:54]: i'm not sure about defaulting deletion policy to abandon, as using this as replacement to standard iac, i would expect to this be deleted by default, and change this to be deleted? Especialy with non data related resources? i would accept this in such think a bucket / cloudsql instance, as this also works similar in terraform, but with api?

File: charts/project-config-connector/Chart.yaml (hunk 1 above)
Comment (karolcienkosz) [reply charts/project-config-connector/Chart.yaml]: fixed

File: charts/project-config-connector/templates/project.yaml (hunk 2 above)
Comment (karolcienkosz) [reply charts/project-config-connector/templates/project.yaml:54]: for me, it is 50/50. lack of abandon is risky and can leads to accidental downtime of some services. in 90% cases when you enable some API, you forget about it

**Review by michalbagrowski — APPROVED** [review]

**Review by mirkoszy — APPROVED** [review]

**Pull request merged**